routing/router.go

```diff
@@ -563,6 +563,14 @@ func (r *ChannelRouter) Start() error {
 			}
 		}
 
+		// The graph pruning might have taken a while and there could be
+		// new blocks available.
+		bestHash, bestHeight, err = r.cfg.Chain.GetBestBlock()
```

Comment:
Wouldn't the behavior that causes us to send the latest block hash w/ a new epoch subscription address things?

Reply:
If I understand your connection correctly, I think no. We're using the `BitcoindFilteredChainView` (or whatever backend) here which only gives us the block from the moment of `Start()` (router.go#L535). So either we start the chain view earlier or use this workaround. I don't really know this code well enough to make an informed decision.

@joostjager what's your take on this?